style.css

```diff
@@ -415,6 +420,65 @@ hr.mobile {
   display: none;
 }
 
+.modal-mobile,
+.modal-desktop {
+  position: fixed;
+  top: 0;
+  background-color: #fff;
+  z-index: 999;
+  padding: 5px 12px 24px 12px;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  width: 100%;
+
```

Comment:
- [ ] You did a great job on the styles but for this project, you need to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=Je0IY5iv4NIVovwd-0) template as much as possible. Please fix the below issues and make sure your design matches the Figma design.
  - Please increase the height of the model and increase the gap between project content.
  - Please make sure the model background is blurred.
  - Please increase font weight for project technologies.
  - Please make sure the model buttons are aligned to the bottom of the model as on the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/209132034-7fa76b09-4f12-4d58-bb35-5a950e10bd00.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/209130340-c4a961f0-df52-4b66-bdec-4aa4656457f6.png)